habitat/tasks/rearrange/rearrange_grasp_manager.py

```diff
@@ -228,6 +227,9 @@ def snap_to_obj(self, snap_obj_id: int, force: bool = True) -> None:
             )
 
         self._snapped_obj_id = snap_obj_id
+        if force:
+            # Set the transformation to be in the robot's hand already.
+            self.snap_rigid_obj.transformation = self._sim.robot.ee_transform
```

Comment:
Seems redundant

Reply:
No, this is important to keep since some functionality is needed to automatically snap the object into the robot's hand, even if the object is far away.

Reply:
It is my fault. I went too ast and did not realize the block 
```
        if force:
            # Set the transformation to be in the robot's hand already.
            self.snap_rigid_obj.transformation = self._sim.robot.ee_transform
```
was added twice (once on main and once on hab_suite_dev) resulting in literal redundant/duplicated code.